pkg/koordlet/statesinformer/states_informer.go

```diff
@@ -194,7 +194,8 @@ func (s *statesInformer) GetNode() *corev1.Node {
 	nodeInformerIf := s.states.informerPlugins[nodeInformerName]
 	nodeInformer, ok := nodeInformerIf.(*nodeInformer)
 	if !ok {
-		klog.Fatalf("node informer format error")
+		klog.Errorf("node informer format error")
+		return nil
```

Comment:
`return` is enough

Reply:
The `GetNode() *corev1.Node` declaration in the StatesInformer interface returns' *corev1.Node ', so 'return nil' is required here.

```
type StatesInformer interface {
	GetNode() *corev1.Node
	GetNodeSLO() *slov1alpha1.NodeSLO
	GetAllPods() []*PodMeta
	GetNodeTopo() *topov1alpha1.NodeResourceTopology
}
```

Reply:
you're right!